.travis.yml

```diff
@@ -1,17 +1,18 @@
 language: bash
```

Comment:
Is there a good reason to not change the language here to `ruby` and get the RVM install behavior for free?

Reply:
That's fine by me, too!

Reply:
Since it's already merged, I'd say it's up to you if you feel it's worth it.